inference-engine/src/mkldnn_plugin/emitters/jit_snippets_emitters.hpp

```diff
@@ -0,0 +1,573 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/rt_info.hpp>
+#include <ngraph/variant.hpp>
+
+#include "jit_emitter.hpp"
+using namespace Xbyak;
+namespace MKLDNNPlugin {
+
+#define SNIPPETS_MAX_SNIPPETS_DIMS 7
+#define SNIPPETS_MAX_HARNESS_DIMS 5
+#define SNIPPETS_MAX_TILE_RANK 2
+#define GET_OFF(field) offsetof(jit_snippets_const_args, field)
+struct jit_snippets_const_args {
```

Comment:
Const because they are set only once before the parallel_for (as opposed to indexes which are set inside the parallel for), that was the idea. Call is also good, let's go with it then.